common/s3util.cpp

```diff
@@ -273,7 +273,8 @@ shared_ptr<S3Util> S3Util::BuildS3Util(
           read_ratelimit_mb * 1024 * 1024);
   SDKOptions options;
   Aws::InitAPI(options);
```

Comment:
I don't think we have it before we use the global client.
InitAPI() was introduced by the new aws library, which was added by Connell after we switched to the global client.
From AWS page https://aws.amazon.com/cn/blogs/developer/aws-sdk-for-c-simplified-configuration-and-initialization/  it's more like a global initialization function.